src/js/get_profile_data.js

```diff
@@ -148,10 +149,16 @@
       }
 
       aliasLabelInput.dataset.label = newAliasLabel;
-      setTimeout(()=> {
+      setTimeout(() => {
         aliasLabelWrapper.classList.remove("show-saved-confirmation");
       }, 1000);
 
+    };
+    aliasLabelInput.addEventListener("focusout", saveAliasLabel);
+    aliasLabelForm?.addEventListener("submit", (event) => {
```

Comment:
Does this `?` throw an error? 

```suggestion
    aliasLabelForm.addEventListener("submit", (event) => {
```

Reply:
No, the expression evaluates to `undefined` if `aliasLabelForm` is `undefined`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

In other words, it's basically equivalent to this:

    (typeof aliasLabelForm !== "undefined" && aliasLabelForm !== null) ? aliasLabelForm.addEventListener(/* ... */) : undefined;

Reply:
That's two different TILs. Awesome job! 🚀